### PR TITLE
Moved spock-shaded to ddf-support

### PR DESCRIPTION
**Pull request by rzwiefel**
#### Who is reviewing it? 
anyone
#### How should this be tested? (List steps with links to updated documentation)
build, ensure coverages still met and spock tests still running
#### Any background context you want to provide?
spock should be a part of the ddf-support project since it has no direct relevance or use being a child of the ddf project. 
#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1012,6 +1012,11 @@ are essentially embedding a slightly modified version of felix -->
             <version>1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ddf.support</groupId>
+            <artifactId>spock-shaded</artifactId>
+            <version>${ddf.support.version}</version>
```
Comment (coyotesqrl) [inline pom.xml]: ❓ Should be scope-test, yes?

**Comment by bdeining** [comment]
This needs a ticket number, and no one is tagged to review it

File: pom.xml (hunk 1 above)
Comment (rzwiefel) [reply pom.xml]: Yes, fixed

**Comment by paouelle** [comment]
Can we also wait before merging this for my PR to complete as I have direct dependencies and changes to the Spock extensions?

**Comment by rzwiefel** [comment]
This will happen later in life - closing for now

**Comment by paouelle** [comment]
It is currently happening but it is moving to codice-test instead

**Pull request closed without merging**